README.md

```diff
@@ -1,14 +1,35 @@
 ![Cover logo](./cover.svg)
 
-# instant-segment: fast English word segmentation in Rust
+# Instant Segment: fast English word segmentation in Rust
 
 [![Documentation](https://docs.rs/instant-segment/badge.svg)](https://docs.rs/instant-segment/)
 [![Crates.io](https://img.shields.io/crates/v/instant-segment.svg)](https://crates.io/crates/instant-segment)
+![PyPI](https://img.shields.io/pypi/v/instant-segment)
 [![Build status](https://github.com/InstantDomainSearch/instant-segment/workflows/CI/badge.svg)](https://github.com/InstantDomainSearch/instant-segment/actions?query=workflow%3ACI)
 [![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE-APACHE)
 
-instant-segment is a fast Apache-2.0 library for English word segmentation.
-It is based on the Python [wordsegment][python] project written by Grant Jenkins,
+```python
```

Comment:
I think we should have a a one or two sentence description of what it actually is before the examples, and probably introduce the examples with an `Examples` heading of some kind.